### PR TITLE
Chore/separate auth rate limits into its own page

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: studio/components/interfaces/Auth/RateLimits/RateLimits.tsx [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+                }
+              >
+                <FormSectionContent loading={false}>
+                  <FormField_Shadcn_
+                    control={form.control}
+                    name="RATE_LIMIT_TOKEN_REFRESH"
+                    render={({ field }) => (
+                      <FormItem_Shadcn_>
+                        <FormControl_Shadcn_>
+                          <Input_Shadcn_ disabled type="number" {...field} />
```
Comment (alaister) [inline studio/components/interfaces/Auth/RateLimits/RateLimits.tsx:280]: Why is this disabled?

File: studio/components/interfaces/Auth/RateLimits/RateLimits.tsx [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+                }
+              >
+                <FormSectionContent loading={false}>
+                  <FormField_Shadcn_
+                    control={form.control}
+                    name="RATE_LIMIT_VERIFY"
+                    render={({ field }) => (
+                      <FormItem_Shadcn_>
+                        <FormControl_Shadcn_>
+                          <Input_Shadcn_ disabled type="number" {...field} />
```
Comment (alaister) [inline studio/components/interfaces/Auth/RateLimits/RateLimits.tsx:315]: Same with this one?

File: studio/components/interfaces/Auth/RateLimits/RateLimits.tsx [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+                </div>
+              }
+            >
+              <FormSection
+                id="email-sent"
+                header={
+                  <FormSectionLabel
+                    description={
+                      <p className="text-foreground-light text-sm">
+                        How many emails can be sent per hour
```
Comment (alaister) [inline studio/components/interfaces/Auth/RateLimits/RateLimits.tsx:147]: I've always wondered if this is a global limit or per user? Might be worth updating this message accordingly

File: studio/components/interfaces/Auth/RateLimits/RateLimits.tsx (hunk 1 above)
Comment (joshenlim) [reply studio/components/interfaces/Auth/RateLimits/RateLimits.tsx:280]: The 2 fields are not exposed through API yet, hence why they're values are hard coded for now too
Needs some BE work before users can self-serve

File: studio/components/interfaces/Auth/RateLimits/RateLimits.tsx (hunk 2 above)
Comment (joshenlim) [reply studio/components/interfaces/Auth/RateLimits/RateLimits.tsx:315]: https://github.com/supabase/supabase/pull/18265#discussion_r1363577819

File: studio/components/interfaces/Auth/RateLimits/RateLimits.tsx (hunk 3 above)
Comment (joshenlim) [reply studio/components/interfaces/Auth/RateLimits/RateLimits.tsx]: gotcha! updated the descriptions  👍

**Review by alaister — APPROVED** [review]
LGTM!

**Comment by GaryAustin1** [comment]
Got the first question on what this is from a user.  Had to hard refresh the browser to see what they were talking about.
Any info on this.  The question was what is the rate limit for refresh token as it did not appear to do anything and then a discussion about is it per ip or system wide.  How is it related to the rate limits in https://supabase.com/docs/guides/platform/going-into-prod#auth-rate-limits?

**Pull request merged**